.circleci/config.yml

```diff
@@ -323,6 +323,10 @@ workflows:
           type: approval
           requires:
             - build_tag_push
+          filters:
+            branches:
+              ignore:
+                - master
       - deploy_dev:
           requires:
             - hold_dev
```

Comment:
```suggestion
            - hold_dev
          filters:
            branches:
              ignore:
                - master   
```

Reply:
Since we ignore master on deploy_dev I believe this would be redundant